### ISSUE TITLE
role="separator" announces with incorrect role in Chrome with VPC off.

### ISSUE THREAD

**Issue by gdkraus** [FreedomScientific/VFO-standards-support#489]
### Summary

With the VPC off, when focus lands on role="separator", JAWS incorrectly announces it as a menu at first, and only later in its description says it is a "separator"

This is related to the issue raised in #237.

### Expected result

JAWS should announce that it is a separator and should announce any pertinent values set with aria-valuenow

### Actual result

In the example provided below, JAWS says "menu, the great dividing line separator, 15"

### Example

[Code Example](https://codepen.io/gdkraus-sas/pen/oNYXvYM)

### Additional Information

With the VPC on, it correctly announces the role, but it does not announce the aria-valuenow.

#### JAWS version and build number

JAWS: 2021.2012.57
 
#### Operating System and version

Windows 10

#### Browser and version:

Chrome: 88.0.4324.96

**Comment by gdkraus** [comment]
Further, when the aria-valuenow updates, JAWS does not announce updates to the value. The only time a value is announced, with the VPC off, is when you first land on the control.

**Comment by accessiblecohn** [comment]
In Chrome / Edge v101 this is still occurring. with PC cursor active, JAWS/Chome assumes the seperator is a menu seperator. 
Firefox no longer gives incorrect information about this role help description.

I believe that this pattern is being implmented to have multiple panels that can change size. This being the case, the behavior should be similar to a slider.

**Comment by george-cz** [comment]
Hello, this has been open for over 3 years now without any response.

The support for `separator` role is still poor. There are no announcements for `aria-valuenow` and `aria-valuetext` at all and the forms mode is not activated correctly on focus of the separator (it has to be activated with Enter).

These issues are preventing us from using the `separator` role and we had to resort to using `slider`, which has better screen reader support, but is not the appropriate role for the scenario we are building. 

Please let us know if there are any plans for this to be fixed. 

Thanks!
